codec_specs.md

```diff
@@ -812,7 +812,7 @@ Codec ID: S_TEXT/SSA
 Codec Name: Subtitles Format
 
 Description: The [Script Info] and [V4 Styles] sections are stored in the codecprivate. Each event is stored in its own Block.
-For more information, see [the SSA/ASS section in Subtitles](#subtitles).
+For more information; see [the SSA/ASS section in Subtitles](#subtitles).
```

Comment:
This change looks wrong. When in doubt if you should use a comma or a semicolon try using a full stop instead. If the result still makes sense, you use a semicolon. And that's not the case here: "For more information. See URL." is not OK.

I'm not going to add a comment for each of these "For more information, see…" changes, but they're all wrong IMHO.